src/components/diagnosticReportView/DiagnosticReportCard.tsx

```diff
@@ -152,11 +157,26 @@ const DiagnosticReportCard: React.FC<DiagnosticReportCardProps> = ({
       const hazardTableId = unifiedInversionSolutions[currentImage].solution.hazardId;
       hazardTableId ? setHazardId(hazardTableId) : setHazardId('');
       let metaList: MetaArguments = [];
+
       if (sweepArgs) {
         metaList = filteredMetaGT(unifiedInversionSolutions[currentImage].solution.meta, sweepArgs);
       } else if (sweepList) {
         metaList = filterMetaArguments(unifiedInversionSolutions[currentImage].solution.meta, sweepList);
       }
+
+      if (
+        [0, 2].includes(unifiedInversionSolutions[currentImage].type) &&
```

Comment:
@benjamineac I'm curious about this line. might get you to show me this in action